### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by Jet004**
Request to merge all initial code into the main branch

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		"dev": "concurrently \"npx tsc -p tsconfig.build.json --watch\" \"nodemon dist/index.js\"",
+		"start": "node dist/index.js"
+	},
+	"keywords": [],
+	"contributors": [
+		"Jacob Turner",
+		"Brandon Kane",
+		"Lisa Li",
+		"Daisy Yang",
+		"Christovao Galambos",
```
Comment (galarist) [inline package.json]: Name correction: Cristovao Galambos

**Review by galarist — CHANGES_REQUESTED** [review]
package.json meeds to be corrected for the correct contributor name (Cristovao Galambos)

**Review by 2016lisali — APPROVED** [review]

**Review by galarist — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Jet004) [reply package.json]: Fixed and in the pipeline. I'll wait to merge the changes before merging this pull request

**Review by QiujuYang — APPROVED** [review]
Well done！

**Review by B08waffles — APPROVED** [review]

**Pull request merged**